src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -399,24 +409,68 @@ limitations under the License.
                     </PARAM>
                 </PARAM>
           </PARAM>
-       <ACTION builtin="clish_nop"></ACTION>
+       <ACTION>
+         if test "${switchport-subcommands}" = "access"; then&#xA;
+             python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_vlan_interfaces_interface_ethernet_switched_vlan_config ${iface} ACCESS ${vlan-id}&#xA;
+         else&#xA;
+             python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_vlan_interfaces_interface_ethernet_switched_vlan_config ${iface} TRUNK ${vlan-id}&#xA;
+        fi
+       </ACTION>
        </COMMAND>
-       <COMMAND name="sflow" help="interface sFlow settings"/>
-            <COMMAND name="sflow sampling-rate" help="Set sampling-rate" ptype="SUBCOMMAND" mode="subcommand">
-                <PARAM name="rate" help="sampling rate" ptype="UINT"/>
-            	<ACTION builtin="clish_nop"/>
-            </COMMAND>
-            <COMMAND name="sflow enable" help="enable sflow" ptype="SUBCOMMAND" mode="subcommand">
-            	<ACTION builtin="clish_nop"/>
-            </COMMAND>
-       <COMMAND name="no sflow" help="disable sflow"/>
-            <COMMAND name="no sflow sampling-rate" help="set sampling rate to default" ptype="SUBCOMMAND" mode="subcommand">
-                <PARAM name="rate" help="sampling rate" ptype="UINT"/>
-            	<ACTION builtin="clish_nop"/>
-            </COMMAND>
-            <COMMAND name="no sflow enable" help="disable sFlow" ptype="SUBCOMMAND" mode="subcommand">
-            	<ACTION builtin="clish_nop"/>
+
+       <COMMAND
+           name="sflow"
+           help="interface sFlow settings"
```

Comment:
Since you're already editing the sFlow commands, can you capitalize the help strings for these commands?

Reply:
Yes Jeff, I addressed it. These commands were not written in the format we do follow in interface.xml, that's the reason I changed it. Anyway, thanks for that catch :)